<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="calendar" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1120</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>44</y>
      <width>21</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="my_button" name="pushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>0</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Month Event</string>
    </property>
   </widget>
   <widget class="my_checkbox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Fix Calendar</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="my_radio" name="radioButton_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play with Local File</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::YearSection</enum>
    </property>
    <property name="displayFormat">
     <string>yyyy/M/d HH:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save..</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>calendar</class>
   <extends>QCalendarWidget</extends>
   <header>calendar.h</header>
  </customwidget>
  <customwidget>
   <class>my_checkbox</class>
   <extends>QCheckBox</extends>
   <header>mybuttongroup.h</header>
  </customwidget>
  <customwidget>
   <class>my_radio</class>
   <extends>QRadioButton</extends>
   <header>mybuttongroup.h</header>
  </customwidget>
  <customwidget>
   <class>my_button</class>
   <extends>QPushButton</extends>
   <header>mybuttongroup.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
